x-pack/plugins/lens/common/expressions/rename_columns/rename_columns.ts

```diff
@@ -6,10 +6,10 @@
  */
 
 import { i18n } from '@kbn/i18n';
-import type { RenameColumnsExpressionFunction } from './types';
+import type { MapToOriginalColumnsExpressionFunction } from './types';
 
-export const renameColumns: RenameColumnsExpressionFunction = {
-  name: 'lens_rename_columns',
+export const renameColumns: MapToOriginalColumnsExpressionFunction = {
+  name: 'lens_map_to_original_columns',
```

Comment:
What do we think about this expression function name change? I understand that `lens_rename_columns` was meant to be general wording, but it isn't used anywhere else and this name more clearly communicates how the function is actually used IMO.

If it seems better to others as well, I can rename the files.

Reply:
@dej611 any feelings on this one?

Reply:
No strong opinion here.
Just noticed there's a mention of `lens_rename_columns` into `x-pack/plugins/lens/server/migrations/saved_object_migrations.test.ts` to be migrated

Reply:
Good catch on the mentions. But, I'm not sure we should change those expressions since they're in tests for previous Kibana versions. WDYT?